lib/ffmprb.rb

```diff
@@ -19,20 +19,33 @@ module Ffmprb
   class Error < StandardError
   end
 
+  Util.ffmpeg_cmd = ['ffmpeg']
+  Util.ffprobe_cmd = ['ffprobe']
+
+  Process.duck_audio_hi = 0.9
+  Process.duck_audio_lo = 0.1
+  Process.duck_audio_transition_sec = 1
+  Process.duck_audio_silent_min_sec = 3
+  Filter.silence_noise_max_db = -40
+
   Util::IoBuffer.blocks_max = 1024
   Util::IoBuffer.block_size = 64*1024
   Util::IoBuffer.timeout = 9
 
   class << self
 
+    # NOTE the form with the block returns the result of #run
+    # NOTE the form without the block returns the process (before it is run)
     def process(*args, &blk)
```

Comment:
How about 2 different methods process and process! ? The first returns process instance the second the result of #run